src/angular/src/app/pages/settings/integrations.component.html

```
@@ -0,0 +1,123 @@
+      <input type="checkbox" class="form-check-input" [(ngModel)]="form.enabled" />
+      Enabled
+    </label>
+  </div>
+</ng-template>
+
+<div class="integrations">
+  <h3 class="card-header integrations-card-header">
+    <span>Integrations</span>
+    @if (!adding) {
+      <div class="add-buttons">
+        <button class="btn btn-sm btn-add" (click)="onStartAdd('sonarr')">+ Sonarr</button>
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)